src/nvim/decoration.c

```diff
@@ -1,3 +1,6 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check
```

Comment:
Duplicate?

Reply:
Hmm, the script is supposed to detect existing comments.

Reply:
Ah, third line is supposed to be blank, not a comment.